packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -175,7 +182,19 @@ export class SerializedStateManager {
 				await this.refreshLatestSnapshot(supportGetSnapshotApi);
```

Comment:
I do not know if it matters but notice that code above (this.offlineLoadEnabled case) will have all the blobs (i.e. it spent time reading all the blobs), so it can also construct ISnapshot and return it.
It's worth doing this optimization if we believe a combo of old driver + this.offlineLoadEnabled will give us more exposure. I do not think it will, as our limiting factor is not driver changes, but loader changes, so that's probably fine.

But, on that front, if we have all the changes that we needed in all drivers in previous version (let's say RC3), then I think we can start simplifying at least Loader layer and assume that drivers will always return us ISnapshot.

Though we should double check that claim. I remember that was some surprise discussion with Loop where they challenged our understanding on the assumptions in this space, but by now I do not remember what it was.